2d/camera-zoom/UI/Zoom_slider.gd

```diff
@@ -1,20 +1,21 @@
 class_name ZoomSlider
-extends Control
+extends VSlider
+
+signal vslider_changed(new_value)
 
 onready var zoom_camera: ZoomingCamera2D = get_node(
 	"/root/ZoomingCameraDemo/Player/ZoomingCamera2D"
 )
-onready var zoom_vslider: VSlider = $VSlider
 
 
 func _ready() -> void:
-	EVENTS.connect("zoom_level_changed", self, "_on_zoom_level_changed")
-	_on_zoom_level_changed(zoom_camera._zoom_level)
+	zoom_camera.connect("zoom_level_changed", self, "_on_ZoomingCamera2D_zoom_level_changed")
+	_on_ZoomingCamera2D_zoom_level_changed(zoom_camera._zoom_level)
```

Comment:
Don't call a callback function directly. The name says this function should only be called when a given event occurs. Here, the zoom level didn't change.